articles/active-directory/develop/msal-js-sso.md

```diff
@@ -91,7 +91,7 @@ If you don't have SID claim configured or need to bypass the account selection p
 ```javascript
 var request = {
   scopes: ["user.read"],
-  loginHint: preferred_username,
+  login_hint: preferred_username,
```

Comment:
Add @sameerag and @EmLauber to check me on this, but I don't these changes are quite right, @JamesTran-MSFT.

The property name is `loginHint` and the claim name is `login_hint`:

https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_common.html#commonauthorizationurlrequest

> Ps. If we deep-linked to our API reference, this sort of thing would be much easier to discover. *cough*

Reply:
Yes. This isn't right. There are multiple missing pieces in this documentation. 
- First let us get the naming clarified: We use camelcase for the variable names in MSAL JS and we do the needed translation in the query parameters we generate later - and this is internal to MSAL. 
- Second, we accepted `loginHint` as a parameter to the MSAL request interface and it used to be `preferred_username` claim in the idToken.
- Third, there is a new `login_hint` claim support added to the idToken and I think we should add documentation on it. 

@EmLauber please work with me to close this.